browser/browser-ui/src/main/AndroidManifest.xml

```diff
@@ -14,13 +14,10 @@
   ~ limitations under the License.
   -->
 
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
-    android:shape="rectangle">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <size
-        android:width="1dp"
-        android:height="1dp" />
+    <application>
 
-    <solid android:color="?daxColorLines" />
+    </application>
```

Comment:
Do we need this empty manifest or can we delete it?

Reply:
yeah we do for Gradle sync